polkadot/api/src/lib.rs

```diff
@@ -280,49 +277,46 @@ impl<S: state_machine::Backend> ClientBlockBuilder<S>
 {
 	// executes a extrinsic, inherent or otherwise, without appending to the list
```

Comment:
gah. bad comment.

Reply:
https://github.com/paritytech/polkadot/blame/513b544fb505b8c59f07adf10e8dc81de34d2a73/polkadot/api/src/lib.rs#L278

:)

(i remember this from the relevant PR but didn't bring it up then for some reason)